NCS.DSS.Customer/PatchCustomerHttpTrigger/Function/PatchCustomerHttpTrigger.cs

```diff
@@ -66,89 +65,89 @@ public async Task<IActionResult> RunAsync([HttpTrigger(AuthorizationLevel.Anonym
-                log.LogWarning("UResponse Status Code: [{response.StatusCode}]. nable to locate 'APIM-TouchpointId' in request header");
+                var response = new BadRequestObjectResult(HttpStatusCode.BadRequest);
+                log.LogWarning("UResponse Status Code: {StatusCode}. nable to locate 'APIM-TouchpointId' in request header", response.StatusCode);
                 return response;
             }
 
             var ApimURL = _httpRequestHelper.GetDssApimUrl(req);
             if (string.IsNullOrEmpty(ApimURL))
             {
-                var response = new BadRequestObjectResult(400);
-                log.LogWarning("UResponse Status Code: [{response.StatusCode}]. nable to locate 'apimurl' in request header");
+                var response = new BadRequestObjectResult(HttpStatusCode.BadRequest);
+                log.LogWarning("UResponse Status Code: {StatusCode}. nable to locate 'apimurl' in request header", response.StatusCode);
                 return response;
             }
 
-            log.LogInformation($"C# HTTP trigger function Patch Customer processed a request. By Touchpoint {touchpointId}");
+            log.LogInformation("C# HTTP trigger function Patch Customer processed a request. By Touchpoint {touchpointId}",touchpointId);
 
             if (!Guid.TryParse(customerId, out var customerGuid))
             {
                 var response = new BadRequestObjectResult(customerGuid);
-                log.LogWarning($"Response Status Code: [{response.StatusCode}]. Unable to parse 'customerId' to a Guid: {customerId}");
+                log.LogWarning("Response Status Code: {StatusCode}. Unable to parse 'customerId' to a Guid: {customerId}", response.StatusCode,customerId);
                 return response;
             }
 
             var subContractorId = _httpRequestHelper.GetDssSubcontractorId(req);
             if (string.IsNullOrEmpty(subContractorId))
-                log.LogInformation($"Unable to locate 'SubContractorId' in request header");
+                log.LogInformation("Unable to locate 'SubContractorId' in request header");
 
             Models.CustomerPatch customerPatchRequest;
 
             try
             {
-                log.LogInformation($"Attempt to get resource from body of the request");
+                log.LogInformation("Attempt to get resource from body of the request");
                 customerPatchRequest = await _httpRequestHelper.GetResourceFromRequest<Models.CustomerPatch>(req);
             }
             catch (Exception ex)
             {
                 var response = new UnprocessableEntityObjectResult(_dynamicHelper.ExcludeProperty(ex, ["TargetSite"]));
-                log.LogError($"Response Status Code: [{response.StatusCode}]. Unable to retrieve body from req", ex);
+                log.LogError("Response Status Code: {StatusCode}. Unable to retrieve body from req. {Exception}", response.StatusCode, ex.Message);
```

Comment:
The log should pass the exception object:
`log.LogError(ex, "Response Status Code: {StatusCode}. Unable to retrieve body from req. {Exception}", response.StatusCode, ex.Message);`